init.sh

```diff
@@ -1,19 +1,16 @@
-# Restore and build the dotnet app
-dotnet restore
-dotnet build
-
-# Install NPM and start the server, it will run side-by-side with the dotnet API server for speed
+# First initial install of npm to ensure it runs npm
```

Comment:
Tested and confirmed it completed. Did all the initial stuff and then ran ./start.sh to run the server.